examples/milp_factory_schedule.jl

```diff
@@ -0,0 +1,96 @@
+#=
+julia> VERSION
+v"1.1.0"
+
+pkg> status JuMP Cbc GLPK
+  [9961bab8] Cbc v0.6.0
+  [60bf3e95] GLPK v0.9.1
+  [4076af6c] JuMP v0.19.0
+
+This is a Julia translation of part 5 from "Introduction to to Linear Programming with Python" 
+available at https://github.com/benalexkeen/Introduction-to-linear-programming
+
+=#
```

Comment:
Please copy the standard license header from another file.

Reply:
added license header, although I am not sure if I need to add any additional info since I am not original author of python example I linked to.